### DISCUSSION TITLE
Can't import Spacy after clean install

### DISCUSSION THREAD

**Discussion by diggymacd** [explosion/spaCy#8966] [Help: Installation] [answered]
Unable to import spacy after apparently successful download with pip.  I get the following error:

```
  File "C:\Users\diggy\AppData\Local\Programs\Python\Python38\Lib\site-packages\srsly\ujson\__init__.py", line 1, in <module>
    from .ujson import decode, encode, dump, dumps, load, loads  # noqa: F401
ModuleNotFoundError: No module named 'srsly.ujson.ujson'
```

This is using Spyder 5.1.1, with Python 3.7.9 (64 bit), Windows 10.  I have tried updating the various modules called out in the Spacy load, and used pip 'check' on all modules.  Pip thinks everything is fine, but it won't import.

Any hep would be appreciated; thanks in advance.

**Comment by polm** [comment, marked as answer]
Sorry you're having trouble with this. 

Recently someone had the same error because their installs of Thinc and spaCy were mixed between conda and non-conda Python somehow, you can see that issue here: https://github.com/explosion/spaCy/discussions/8199. 

Just to see if it's an issue with your environment, could you try installing spaCy in vanilla Python without using Spyder/Anaconda? Based on the path in your error it looks like srsly is installed in your vanilla Python and not an Anaconda environment.

It also might help to check in your srsly install and see what `pyd` files you have there - you should have one for ujson, but it might not be there is something went wrong with the install.

**Comment by diggymacd** [comment]
That fixed the problem; thanks!

Turns out it was some obscure Spyder configuration that wasn't finding 
the ujson module.  As soon as I ran the code in native Python, 
everything worked.  I'm going to switch IDE to something else.

Doug

------ Original Message ------
From: "polm" ***@***.***>
To: "explosion/spaCy" ***@***.***>
Cc: "diggymacd" ***@***.***>; "Author" 
***@***.***>
Sent: 8/15/2021 4:42:55 AM
Subject: Re: [explosion/spaCy] Can't import Spacy after clean install 
(#8966)

**Reply by polm** [reply]
Glad to hear you figured it out, thanks for reporting back!
